### DISCUSSION TITLE
Security Onion no longer generates alerts and gives a 404 error for kibana after power outage

### DISCUSSION THREAD

**Discussion by kgoode517** [Security-Onion-Solutions/securityonion#7809] [Unsupported Versions] [answered]
Hello folks, I am at a complete loss at why my virtual box standalone  security onion no longer generates alerts and is giving me a 404 error for kibana after I lost power due to a storm. I run so-status and get back everything as running as okay. I run sudo docker exec -u 0 -it so-wazuh var/ossec/bin/agent_control -lc and see all of my wazuh agents are connected. I updated with soup to see if that would fix the issue but no such luck. I've restarted several times with no change. What can I use to troubleshoot the problem from here?

**Comment by dougburks** [comment]
Have you checked the logs in `/opt/so/log/`?

**Comment by kgoode517** [comment]
@dougburks  Thank you for taking a look. No I did not and I am assuming my focus should be on Suricata and Kibana logs going into their directories show blank logs for both kibana.log and suricata.log even when I unzip for the dates.

**Comment by dougburks** [comment, marked as answer]
Next step would be to check Docker logs for each of those:
```
sudo docker logs so-kibana
```
```
sudo docker logs so-suricata
```

**Reply by kgoode517** [reply]
Thank you for explaining that. That helped me tremendously. I found the following Kibana error and I'm trying to figure out how to resolve it but I don't really see a clear way to fix it.  FATAL  Error: Unable to complete saved object migrations for the [.kibana_task_manager] index: Unable to complete the OUTDATED_DOCUMENTS_SEARCH_OPEN_PIT step after 15 attempts, terminating. I've tried sudo salt-call state.apply kibana.so_savedobjects_defaults -l info queue=True but that didn't help. 
Suricuta logs don't seem to have any errors associated with them.

**Reply by kgoode517** [reply]
This seems to incorporate issue https://github.com/Security-Onion-Solutions/securityonion/discussions/2764. This is definitely not the first time this has happened to me and although I can just recreate the server  as a workaround solution I would rather not have to rely on that anytime the server shuts down unexpectedly because it will just break again. This issue is reproducible time and again by unexpected shutdowns of the server which never recovers the functionality.

**Reply by Acewiza** [reply]
Critical infrastructure components are normally run on uninterruptible power supplies in order to avoid this type scenario.

**Reply by kgoode517** [reply]
That’s a great point and $60 is a small price to pay for that. Thank you I didn’t even consider it as a solution.

**Reply by dougburks** [reply]
For others that may find this discussion, our documentation recommends a UPS here:
https://docs.securityonion.net/en/2.3/hardware.html#ups
